### ISSUE TITLE
Couchpotato just deleting downloaded files

### ISSUE THREAD

**Issue by darkdelphi** [clinton-hall/nzbToMedia#1264]
Suddenly Couchpotato has started to just delete movies downloaded. Sab shows successful postprocessing but the movie is just gone. I can't seem to find the problem or exactly where to search to resolve this. 
Help me Obi-won Kenobis, your my only hope. 
 [17:03:39] [INFO]::MAIN: Loading config from [/usr/local/share/nzbToMedia/autoProcessMedia.cfg]
[17:03:39] [INFO]::MAIN: Checking database structure...
[17:03:39] [INFO]::MAIN: Checking if git needs an update
[17:03:40] [INFO]::MAIN: No update needed
[17:03:40] [INFO]::MAIN: nzbToMedia Version:80c8ad58523ab99825c02f3855f9bd3dc9945d57 Branch:master (FreeBSD 10.3-RELEASE)
[17:03:40] [INFO]::MAIN: #########################################################
[17:03:40] [INFO]::MAIN: ## ..::[nzbToMedia.pyc]::.. ##
[17:03:40] [INFO]::MAIN: #########################################################
[17:03:40] [INFO]::MAIN: Script triggered from SABnzbd
[17:03:41] [INFO]::MAIN: Auto-detected SECTION:CouchPotato
[17:03:41] [INFO]::MAIN: Calling CouchPotato:movie to post-process:REDACTED.nzb
[17:03:41] [INFO]::MAIN: Attemping imdbID lookup for REDACTED.nzb
[17:03:41] [INFO]::MAIN: Searching folder and file names for imdbID ...
[17:03:41] [INFO]::MAIN: Searching IMDB for imdbID ...
[17:03:42] [WARNING]::MAIN: Unable to find a imdbID for REDACTED.nzb
[17:03:43] [INFO]::TRANSCODER: Checking [REDACTED.avi] for corruption, please stand by ...
[17:03:43] [INFO]::TRANSCODER: SUCCESS: [REDACTED.avi] has no corruption.
[17:03:43] [POSTPROCESS]::COUCHPOTATO: Starting renamer scan for REDACTED.nzb
[17:03:43] [POSTPROCESS]::COUCHPOTATO: SUCCESS: Finished renamer scan for folder /mnt/media/downloads/sabnzbd/complete/movie/REDACTED
[17:03:43] [POSTPROCESS]::COUCHPOTATO: Checking for status change, please stand by ...
[17:03:45] [POSTPROCESS]::COUCHPOTATO: SUCCESS: Movie None has now been added to CouchPotato
[17:03:45] [INFO]::CLEANDIR: Doing Forceful Clean of /mnt/media/downloads/sabnzbd/complete/movie/REDACTED
[17:03:45] [INFO]::MAIN: Deleting /mnt/media/downloads/sabnzbd/complete/movie/REDACTED
[17:03:45] [INFO]::MAIN: The /usr/local/share/nzbToMedia/nzbToCouchPotato.py script completed successfully.
CouchPotato: Successfully post-processed REDACTED.nzb!

**Comment by clinton-hall** [comment]
Check CouchPotato logs at 17:03:43
The question here is why didn't CouchPotato rename/move the files

You can try disabling option `force_clean` in nzbToMedia.

**Comment by Madseason41** [comment]
I have the same issue. Log is pretty much the same as above.

I was on the nightly branch for a reason i don't recall, and I kept having issues with the torrents not being downloaded after completing and post processing. After tinkering with it for a while, I noticed I was on nightly, swapped to master and now Couchpotato movies do no not finish renaming and get deleted.

I will turn off force clean and see what happens...
